src/views/ingresses/details/components/IngressDetailsPageTitle/IngressDetailsPageTitle.scss

```diff
@@ -0,0 +1,3 @@
+.ingress-details-breadcrumbs {
+  margin-bottom: var(--pf-v5-global--spacer--sm);
+}
```

Comment:
You could add an empty line here ;)

Reply:
Done! Thanks for pointing this out! I had actually made that and another small change (adding optional chaining operators) and forgotten to push it. Seeing this comment lead me to realize that so I could push the changes.